### ISSUE TITLE
Multi-Topic subscription across tenant and namespace in Java client

### ISSUE THREAD

**Issue by prakashsaraf7** [apache/pulsar#3906]
The document suggest that we can create a multi-topic subscription across different namespaces. However, in MultiTopicConsumerImpl.java - there are explicit check to make sure all topics belong to one namespace only.
Can you please clarify, if it is possible to create multi-topic subscription across tenant and namespace? Or is it documentation bug?

**Comment by skyrocknroll** [comment]
@prakashsaraf7 You can create multiple producers one per namespace and consume.

**Comment by prakashsaraf7** [comment]
So, if I understand correctly, we will have to create multiple consumers, one per namespace and use it for processing? In that case, I suggest correcting the documentation - where it indicates, I can have one consumer for multiple topics spanning across namespaces.

**Comment by aahmed-se** [comment]
Where in the documentation is this mentioned ?

**Comment by prakashsaraf7** [comment]
In https://pulsar.apache.org/docs/en/client-libraries-java/
Multiple-Topic Subscriptions section -

_"You can also subscribe to an explicit list of topics (across namespaces if you wish):"_

**Comment by ypt** [comment]
+1. I also just ran into this. Reading the docs, I had assumed that I was able to create a consumer that could consume topics across different namespaces, but in trying to do so with the Java client, I'm seeing an `java.lang.IllegalArgumentException: Topics should have same namespace` exception.

For what it's worth, my usage looks something like the following:

```kt
client.client.newConsumer()
    .topics(listOf(
        "persistent://tenant/namespace-1/topic",
        "persistent://tenant/namespace-2/topic"
    ))
```
